### PR TITLE
Home styling

### PR DESCRIPTION
**Pull request by adamsjr8576**
#### What does this PR do?
- This PR adds some styling to the Header, MoviesContainer, MovieCard components. 
- formats cards in container using flexbox
- adds inline styling to header and cards to dynamically add the backgroundImage
- creates functionality to randomly add a movie poster to background header on page load
- implements functionality to dynamically add different tomatillos based on the average rating
 - 0-3: brown tomatillo
 - 4-6: yellow tomatillo
 - 7-10: green tomatillo

#### How to test?
- pull down the changes and run `npm start` - refresh to make sure header image changes and color tomatillos match averageRating

#### Notes
- I used inline styling so that backgroundImage has access to the images within the component

#### Screenshots:

Screen Shot 2019-12-18 at 9 08 15 AM

@ckoga 
@JamesRexMiller4

### CODE REVIEW

File: rancid-tomatillos/src/Components/Header/Header.js [hunk 1]
```diff
@@ -1,16 +1,23 @@
 import React from 'react';
 import './Header.scss';
+import { connect } from 'react-redux';
 
-const Header = () => {
+
+const Header = ({ data }) => {
+  const index = Math.floor(Math.random() * Math.floor(20));
```
Comment (JamesRexMiller4) [inline rancid-tomatillos/src/Components/Header/Header.js:7]: Love this functionality, nice work!

File: rancid-tomatillos/src/Components/Header/Header.js [hunk 2]
```diff
@@ -1,16 +1,23 @@
...
   return (
-    <header className='header-main'>
+    <header className='header-main' style={{backgroundImage: `url(${data.movies[index].backdrop_path})`}}>
       <h1 className='header-h1'>Rancid Tomatillos</h1>
       <section className='header-btn-section'>
-        <button className='header-login-btn'>Log In</button>
-        <button className='header-home-btn'>Home</button>
+        <button className='header-btn'>Log In</button>
```
Comment (JamesRexMiller4) [inline rancid-tomatillos/src/Components/Header/Header.js:12]: At some point we will want to dynamically update the inner html of this button, and so we should give it an id to be able to grab it

File: rancid-tomatillos/src/Components/MovieCard/MovieCard.js [hunk 3]
```diff
@@ -1,15 +1,29 @@
...
 const MovieCard = ({ id, poster, avgRating }) => {
+  let icon;
+  if (avgRating < 4) {
+    icon = images.badTomatillo;
+  } else if (avgRating > 3 && avgRating < 7) {
+    icon = images.okayTomatillo;
+  } else if (avgRating > 6) {
+    icon = images.goodTomatillo;
+  }
   return (
```
Comment (JamesRexMiller4) [inline rancid-tomatillos/src/Components/MovieCard/MovieCard.js:14]: yess love this functionality as well. The progressively rancid tomatillos to indicate difference in rating is great! And I think it makes for a better UX than having to make a selection between 1-10

**Review by JamesRexMiller4 — APPROVED** [review]
Everything looks good! Way to go on cranking this out after a long night!

**Review by ckoga — APPROVED** [review]

**Pull request merged**